src/systems/userinput/devices/app-aware-touchscreen.js

```diff
@@ -105,7 +114,7 @@ export class AppAwareTouchscreenDevice {
           // If grab was being delayed, we should fire the initial grab and also delay the unassignment
           // to ensure we write at least two frames with the grab down (since the action set will change)
           // and otherwise we'd not see the falling xform.
-          if (assignment.framesUntilGrab >= 0) {
+          if (assignment.framesUntilGrab > 0) {
```

Comment:
I actually don't have a good idea of why this fix was needed (and if it was even a regression). Without it I noticed that sometimes objects would stick to the cursor and never be dropped)